README.md

```diff
@@ -3,12 +3,22 @@
 [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/busyorg/busy/new-design/LICENSE)
 [![Busy channel on Discord](https://img.shields.io/badge/chat-discord-738bd7.svg)](https://discord.gg/G95rNZs)
 
-# Busy
+# Busy 🚀
 
-Blockchain-based social network where anyone can earn rewards 🚀
+Busy is an Open Source social network and communications platform which extends itself to a variety of rich features and functionality including free digital payments and a marketplace for goods and services. 
```

Comment:
This description is little bit outdated

Reply:
Could you suggest any changes ?

Reply:
we're in the middle of a pivot strategy and redefining it so i don't think it's the best timing for us to update this text for now. I agree it's a bit outdated, but taking position with a new one that is not yet properly defined will probably not help us. I think we need at least few more weeks to have our new strat defined.

For now i'd correct it with something like that:

Busy.org is the next-gen social network and communications platform that includes a variety of rich features and functionalities, allowing content producers, voters, and commentators to get rewarded for their contribution.